src/cf-layout/src/cf-layout.less

```diff
@@ -665,18 +665,22 @@
     &_heading {
         .heading-1();
 
-        .respond-to-max(800px, {
+        .respond-to-max(@bp-sm-max, {
             .heading-2();
         });
     }
 
-    &_subhead {
+    &_subhead,
+    &_cta {
```

Comment:
It's just a link either way https://github.com/cfpb/cfgov-refresh/blob/flapjack/cfgov/jinja2/v1/_includes/molecules/hero.html#L69-L76